Client/src/Pages/Auth/Register/StepTwo/index.jsx

```
@@ -0,0 +1,104 @@
+						onSubmit={onSubmit}
+						mb={theme.spacing(5)}
+					>
+						<Field
+							type="email"
+							id="register-email-input"
+							label="Email"
+							isRequired={true}
+							placeholder="jordan.ellis@domain.com"
+							autoComplete="email"
+							value={form.email}
+							onInput={(e) => (e.target.value = e.target.value.toLowerCase())}
```

Biome

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)